.github/workflows/react.yml

```diff
@@ -2,13 +2,9 @@ name: React Tests
 
 on:
   push:
-    branches:
-      - "main"
     paths:
       - "react/**"
   pull_request:
-    branches:
-      - "main"
```

Comment:
I realize this is a questionable move on my part to deviate from the preset workflow for this assessment. I would normally not do something like this on my own without consulting the team or other staff on the job. I am generally used to merging my working branches into main instead of pushing main directly. I could have merged my working branch into main and pushed that instead, which would have avoided these changes. I hope this will not penalize me.